backend/main.py

```
@@ -941,17 +1013,25 @@ async def ingest_project_detail(project_id: str, request: ProjectDetailIngestReq
+            status_code=202,
+            content={"message": "Project detail digest initiated asynchronously."}
         )
-        return {"status": "saved", "chars": len(final_text), "mode": (request.mode or "merge").lower()}
+        
     except HTTPException:
         raise
     except Exception as e:
-        logger.error(f"Error ingesting project detail: {e}")
-        raise HTTPException(status_code=500, detail=f"Failed to ingest project detail: {str(e)}")
+        logger.error(f"Error initiating project detail digest: {e}")
+        raise HTTPException(status_code=500, detail=f"Failed to initiate project detail digest: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)